commands/host.vegman

```diff
@@ -224,6 +224,53 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+# @restrict cmd_config tech,admin
+# @doc cmd_config
+# Host config backup and restore
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_default tech,admin
+# @restrict cmd_config_default tech,admin
+# @doc cmd_config_default
+# Restore the factory default Host config
+function cmd_config_default {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to defaults on next boot"
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+  
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
+}
+
+# @sudo cmd_config_nodefault tech,admin
+# @restrict cmd_config_nodefault tech,admin
+# @doc cmd_config_nodefault
+# Undo if the factory default Host was set
+function cmd_config_nodefault {
```

Comment:
I would make this function and status as subcommands (options) of cmd_config_default, like:
> host config default --cancel
> host config default --status